Cargo.toml

```diff
@@ -16,12 +16,13 @@ util = { package = "webrtc-rs-util", version = "0.1.2" }
 lazy_static = "1.3.0"
 tokio = { version = "1.0", features = ["full"] }
 log = "0.4"
-
+net2 = "*"
```

Comment:
looks like net2 is Deprecated, and  are encouraged to consider socket2 instead, according to https://crates.io/crates/net2

Reply:
Thanks. Will implement changes.